dialogflow_v2/__init__.py

```diff
@@ -38,6 +38,13 @@
     )
     warnings.warn(message, DeprecationWarning)
 
+raise RuntimeError(
+    "'dialogflow' has been renamed to 'google-cloud-dialogflow'. "
+    "'dialogflow' will no longer be updated. "
+    "For help upgrading to google-cloud-dialogflow>=2.0.0, see "
+    "https://github.com/googleapis/python-dialogflow/blob/master/UPGRADING.md"
+)
```

Comment:
Users could use this library in production systems and my thought is that it's better not to introduce a breaking changes solely for the purposes of communicating that we are no longer updating this library. We could provide this communication to users via a warning message, and via the README/release notes of this repo. The onus would be on the application maintainers to check for any warnings in their production applications and also periodically check the README and the release notes.

Reply:
Update: we now plan to first release a version that warns, wait 90 days, then release a version that errors. See [doc](https://docs.google.com/document/d/1isYCZFM0zigFwOLzWdKWfc1TJkvOcAXaHlauzUpvY2M/edit#) for details.